picr22-enne-1-voistlust/segment_module/setup.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+# -*- coding: UTF-8 -*-
+# sudo python setup.py install
+
+from distutils.core import setup, Extension

```

Comment:
Are you using this file?

Reply:
didnt use any files in picr22-enne-1-voistlust so removed it